client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -83,8 +84,15 @@ export const getTask = (
 
 			// Change the task title for podcasting sites.
 			if ( isPodcastingSite ) {
-				taskData.title = translate( 'Welcome to your podcast site!' );
+				taskData.title = isLaunchedSite
+					? translate( 'You launched your podcast site!' )
+					: translate( 'Welcome to your podcast site!' );
 				taskData.hideLabel = true;
+				taskData.description = isLaunchedSite
+					? translate(
+							'Now that you’ve created your site, we’ll guide you through completing a few additional steps to continue building your site.'
+					  )
+					: translate( "Next, we'll guide you through setting up and launching your site." );
```

Comment:
Can this use single quotes and an apostrophe like the line above for consistency?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/50056/commits/6ebdc298bed34d99c40f7d60a64245690aec8dab